### ISSUE TITLE
Breakpad support?

### ISSUE THREAD

**Issue by gawen** [getsentry/sentry-native#112]
I see the Breakpad backend had been removed and [is being bring back](https://github.com/getsentry/sentry-native/tree/feat/breakpad-backend).

1. Why had it been removed?
2. What's missing in branch `breakpad-backend` to be merged in `master`? How can I help?

**Comment by jan-auer** [comment]
Hi @gawen, that's perfect timing. I've just recently started to work on getting it back in.

We removed it since we made changes to how the SDK worked internally and weren't able to finish it in time. What's missing now is just proper handling for attachments. There are two open items:
 1. Meta data for attachments need to be persisted on disk. 
 2. Attachments need to be copied alongside the event.

For Android, we've added [`Envelopes`](https://github.com/getsentry/sentry-native/blob/58835e95bac90de369f6a41b8074c7a557663397/src/transports/envelopes.hpp) that make it easier to store multiple files alongside an event. The only thing remaining now is to also use Envelopes to store minidumps + their attachments.
